scripts/konflux/fetch-vuln-feed-data.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+if [[ "$#" < 1 ]]; then
+  echo "Please pass target directory."
+  exit 1
+fi
+
+TARGET_DIR=$1
+
+if [[ ! -d "$TARGET_DIR" ]]; then
+  echo "$TARGET_DIR is not a valid directory"
+  exit 1
+fi
+
+blobs=(
+  nvd-definitions.zip
+  k8s-definitions.zip
+  repo2cpe.zip
+  genesis_manifests.json
+)
+
+for blob in "${blobs[@]}"; do
+  echo "Downloading https://storage.googleapis.com/definitions.stackrox.io/scanner-data/latest/${blob} > $TARGET_DIR/blob-${blob}"
+  curl --fail -s --show-error --retry 4 --retry-max-time 30 --retry-connrefused \
```

Comment:
In order to fully close on the comment [https://github.com/stackrox/scanner/pull/1334#discussion_r1469457260], I suggest adding a TODO comment here as another reminder.

```suggestion
  # TODO(ROX-22130): Assign proper suffix for tagged commits instead of /latest/.
  curl --fail -s --show-error --retry 4 --retry-max-time 30 --retry-connrefused \
```